src/components/RedcapExport/RedcapExport.tsx

```diff
@@ -0,0 +1,236 @@
+    const idx = crfAttribute.findIndex(({ id }) => id === item.id)
+    const tmp = [...crfAttribute]
+    idx >= 0 && tmp.splice(idx, 1, item)
+    setCrfAttribute(tmp)
+  }
+  const classes = useStyles(crfAttribute)
+
+  return (
+    <Dialog
+      fullWidth={true}
+      maxWidth={'lg'}
+      onClose={props.onClose}
+      aria-labelledby="simple-dialog-title"
+      open={props.open}
+      classes={{ paper: classes.dialogPaper }}
+    >
+      <DialogTitle id="simple-dialog-title" disableTypography>
+        <Typography variant="h1">Exporter la cohorte</Typography>
+      </DialogTitle>
+      <DialogContent>
+        <Grid container direction="column" spacing={4}>
+          <Grid item>
+            <div>
+              <Typography variant="h5" gutterBottom>
+                Mode d'export
+              </Typography>
+              <Select value={anonymization} variant="outlined" onChange={handleAnonymization}>
+                <MenuItem value={0} disabled>
+                  Nominatif
+                </MenuItem>
+                <MenuItem value={1} disabled>
+                  Anonymisé
+                </MenuItem>
+                <MenuItem value={2} disabled>
+                  Pseudonymisé
+                </MenuItem>
+              </Select>
+            </div>
+          </Grid>
+          <Grid item>
+            <div>
+              <Typography variant="h5" gutterBottom>
+                Sélectionner les variables à exporter
+              </Typography>
+              <TableContainer component={Paper}>
+                <Table aria-label="simple table">
+                  <TableHead>
+                    <TableRow>
+                      <TableCell align="left">
+                        <Tooltip title="Nom de l'attribut à exporter">
+                          <span>Variable à exporter </span>
+                        </Tooltip>
+                      </TableCell>
+
+                      <TableCell align="left" padding="none">
+                        <Tooltip title="Rennomer la colonne pour l'export">
+                          <span> Nom de colonne (personnalisable) </span>
+                        </Tooltip>
+                      </TableCell>
+                      <TableCell align="right"></TableCell>
+                    </TableRow>
+                  </TableHead>
+                  <TableBody>
+                    {crfAttribute.map((item, index) => {
+                      return <CohortItem item={item} key={index} onDelete={removeItem} onChange={handleChangeItem} />
+                    })}
+                  </TableBody>
+                </Table>
+              </TableContainer>
+              <Button
+                className={classes.addButton}
+                variant="outlined"
+                onClick={handleAddButton}
+                startIcon={<AddIcon />}
+              >
+                Ajouter une variable
+              </Button>
+            </div>
+          </Grid>
+          {/* <Grid item>
```

Comment:
why is this commented out? is this the old version ?

Reply:
Yes, in the resah version there were explaining texts about Redcap export.
Those texts were also refering to a "Champ de recherche" feature that has been removed from the mock, so i just commented it out